src/components/AirTicketsAside/AirTicketsAside.component.js

```diff
@@ -0,0 +1,134 @@
+import { SELECTION_ALL } from 'antd/lib/table/hooks/useSelection';
+import React, { useState, useEffect } from 'react';
+import InputRange from 'react-input-range';
+import 'react-input-range/lib/css/index.css';
+import { connect } from 'react-redux';
+
+import {
+	AirTicketsAsideContainer,
+	AirlinesContainer,
+	TimeContainer,
+	SeatContainer,
+	PriceRangeContainer,
+} from './AirTicketsAside.styles';
```

Comment:
```suggestion
import S from './AirTicketsAside.styles';
```
이렇게 import(<- export default로 빼내야겠죠?) 한 다음

```js
<S.TimeContainer /> 
```
이런 식으로 활용하는 패턴도 있습니다. 이 경우 S는 스타일드 컴포넌트를 값으로 가지고 있는 하나의 객체가 되겠네요. 이 경우 일반 리액트 컴포넌트와 스타일드 컴포넌트를 시각적으로 구분하기 쉽다는 장점도 있겠네요. 참고해주세요!

Reply:
수정했습니다! :smile